src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure implements AreaCalculetion {

```

Comment:
Don't use verbs for class/interface names. The interface name `AreaCalculetion` should be something like `AreaCalculator` as per Java naming conventions.